Naveen-gali/Yelp-App
# Theming Support

### What does this PR do?

- Adds Light and Dark Theme Support.

### Related Issues

- closes #38 

### How to test

- Clone the Repository.
- Switch to feat/theming branch.
- Do yarn to install the dependencies.
- Do yarn run build-stage-ios for ios or yarn run build-stage-android for android.
- Try. Switching the theme in android or ios and you can notice the change in colors.
- Note: As Dark Theme colors are not finalized only Light Theme colors are shown in both themes.

### Screenshots

|Android ScreenShot           |  Ios ScreenShot|
|:-------------------------:|:-------------------------:
| ![Screenshot_1681371427](https://user-images.githubusercontent.com/122969303/231698456-e6f8e7bd-591a-4851-bda8-303ef5b54fa7.png) | ![Simulator Screen Shot - iPhone 14 - 2023-04-13 at 13 46 32](https://user-images.githubusercontent.com/122969303/231698498-ce3d2f74-4b77-456b-ba44-9f5b7e7757d2.png) |

### Checklist

### Additional Notes

- [Yelp Design CookBook](https://www.yelp.com/styleguide)
